app/stable_appconfig.go

```diff
@@ -28,18 +33,51 @@ func setCustomModuleBasics() []module.AppModuleBasic {
 			paramsclient.ProposalHandler, distrclient.ProposalHandler,
 			upgradeclient.ProposalHandler, upgradeclient.CancelProposalHandler,
 		),
+		data.Module{},
+		ecoModule.Module{},
+		group.Module{},
```

Comment:
Data & Group module should still be in experimental app config, as neither of them are intended to be live in mainnet for the upcoming Baikal Upgrade (regen ledger 2.0)

Also, rather than putting ecocredit module in stable, we should remove it from experimental and put it in the normal `app.go` so that it renders in both stable & experimental configs.